document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -15,16 +15,14 @@
 import org.apache.commons.lang3.StringUtils;
 import org.hypertrace.core.documentstore.postgres.Params;
 import org.hypertrace.core.documentstore.postgres.Params.Builder;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class PostgresUtils {
-  private static final Logger LOGGER = LoggerFactory.getLogger(PostgresUtils.class);
   private static final String QUESTION_MARK = "?";
   private static final String JSON_FIELD_ACCESSOR = "->";
   private static final String JSON_DATA_ACCESSOR = "->>";
   private static final String DOT_STR = "_dot_";
   private static final String DOT = ".";
+  public static final String ID_COLUMN = ID;
```

Comment:
Any reason why we need to create another `ID_COLUMN` field? 
Can the `ID` field be used directly, everywhere?

Reply:
I was planning to move all constant at one place. But, can do follow-up.

Reply:
Done.